MAINTAINERS.yaml

```diff
@@ -27,94 +26,73 @@
 #     maintainers:
 #       - John Doe <john.doe@gem5.org>
 #       - Jane Doe <jane.doe@gem5.org>
-#
+#     experts:
+#       - Jack Doe <jack.doe@gem5org>
+#       - Jill Doe <jill.doe@gem5org>
 #
 # The status field should have one of the following values:
 #   - maintained: The component has an active maintainer.
 #   - orphaned: The component is looking for a new owner.
-
-
-pmc:
-  desc: >-
-    PMC Members (general maintainers):
-  status: maintained
-  maintainers:
-    - Andreas Sandberg <andreas.sandberg@arm.com>
-    - Brad Beckmann <bradford.beckmann@gmail.com>
-    - David Wood <david@cs.wisc.edu>
-    - Gabe Black <gabe.black@gmail.com>
-    - Giacomo Travaglini <giacomo.travaglini@arm.com>
-    - Jason Lowe-Power <jason@lowepower.com> (chair)
-    - Matt Sinclair <sinclair@cs.wisc.edu>
-    - Tony Gutierrez <anthony.gutierrez@amd.com>
-    - Steve Reinhardt <stever@gmail.com>
+#
+# The experts field is optional and used to identify people who are
+# knowledgeable about the subsystem but are not responsible for it. Those
+# listed as an expert are typically good to add as a reviewer for pull requests
+# targeting that subsystem.
 
 arch:
   desc: >-
     General architecture-specific components
-  status: maintained
-  maintainers:
-    - Gabe Black <gabe.black@gmail.com>
+  status: orphaned
 
 arch-arm:
   status: maintained
   maintainers:
-    - Andreas Sandberg <andreas.sandberg@arm.com>
     - Giacomo Travaglini <giacomo.travaglini@arm.com>
+    - Andreas Sandberg <andreas.sandberg@arm.com>
 
 arch-gcn3:
   status: maintained
   maintainers:
-    - Matt Poremba <matthew.poremba@amd.com>
     - Matt Sinclair <sinclair@cs.wisc.edu>
+    - Matt Porema <matthew.porema@amd.com>
 
 arch-vega:
   status: maintained
   maintainers:
-    - Matt Poremba <matthew.poremba@amd.com>
     - Matt Sinclair <sinclair@cs.wisc.edu>
+    - Matt Porema <matthew.porema@amd.com>
```

Comment:
I know this is already merged and this isn't super important but my name is spelled incorrectly here including the email

Reply:
Oops!! @ivanaamit can you fix this? It should be "Poremba" (unless I've again made an embarrassing error)